src/chat.rs

```diff
@@ -793,6 +793,13 @@ impl ChatId {
         context
             .sql
             .transaction(|transaction| {
+                let mut stmt = transaction.prepare("SELECT id FROM msgs WHERE chat_id=? AND type=?")?;
+                let rows = stmt.query_map((self, Viewtype::Webxdc), |row| row.get(0))?;
+                // Let's emit events optimistically and not return webxdc `MsgId`s from the
+                // transaction to not add LIMIT to SELECT above.
```

Comment:
I don't understand the comment. what is the meaning of it?

Reply:
I tried to improve the comment. I just don't want to allocate a vector of unlimited size